test/weets/get.spec.js

```diff
@@ -0,0 +1,146 @@
+const supertest = require('supertest');
+const bcrypt = require('bcryptjs');
+const app = require('../../app');
+const sessionsManager = require('../../app/services/sessions_manager');
+const errorsCatalog = require('../../app/schemas/errors_catalog');
+const errors = require('../../app/errors');
+
+const { create: createUser } = require('../factory/users');
+const { create: createWeet } = require('../factory/weets');
+
+const request = supertest(app);
+
+const getWeets = (endpoint, token) =>
+  request
+    .get(endpoint)
+    .set('Authorization', `Bearer ${token}`)
+    .set('Accept', 'application/json')
+    .send();
+
+const hashPassword = async password => {
+  const salt = await bcrypt.genSaltSync(10);
+  const hashedPass = await bcrypt.hashSync(password, salt);
+  return hashedPass;
+};
+
+let rawToken = {};
+let response = {};
+const expiredToken =
```

Comment:
I note that this var is reuse in all spec tests.
So... you can create a common file and exports vars that you use to avoid define it en each test files.